OneSignalSDK/onesignal/src/main/java/com/onesignal/OSInAppMessageController.java

```diff
@@ -94,6 +102,14 @@ protected OSInAppMessageController() {
         );
         if (tempClickedMessageIdsSet != null)
             clickedClickIds.addAll(tempClickedMessageIdsSet);
+
+        if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.JELLY_BEAN_MR2) {
```

Comment:
This if would never be true as we new up a `OSInAppMessageDummyController` instead for per 4.4 devices.

Reply:
true!